### DISCUSSION TITLE
Referencias de como desplegar los estudios en WEB

### DISCUSSION THREAD

**Discussion by afk-mario** [indi-es/estudios#24] [General]

**Comment by afk-mario** [comment]
https://glitch.city/

**Comment by afk-mario** [comment]
http://gamecollectives.com/

**Comment by AlainPlusPlus** [comment]
Como solución temporal podríamos habilitar `GitHub Pages` y agregarle un tema de `Jekyll`, así al menos se vería mejor la presentación actual:
https://docs.github.com/en/github/working-with-github-pages/adding-a-theme-to-your-github-pages-site-with-the-theme-chooser
